kolibri/content/api.py

```diff
@@ -29,12 +32,10 @@ def title_description_filter(self, queryset, value):
         exact_match = queryset.filter(Q(parent__isnull=False), Q(title__icontains=value) | Q(description__icontains=value))
         if exact_match:
             return exact_match
-        # if no exact match, search for non-adjacent match
-
+        # if no exact match, fuzzy search using the stemmed_metaphone field in ContentNode that covers the title and description
         return queryset.filter(
             Q(parent__isnull=False),
-            reduce(lambda x, y: x & y, [Q(title__icontains=word) for word in value.split()]) |
-            reduce(lambda x, y: x & y, [Q(description__icontains=word) for word in value.split()]))
+            reduce(lambda x, y: x & y, [Q(stemmed_metaphone__icontains=doublemetaphone(self.stemmer.stem(word))[0]) for word in value.split()]))
```

Comment:
Keeping both of them on both sides would be maximally fuzzy, since we're doing "OR".